x-pack/legacy/plugins/graph/public/angular/services/saved_workspace_references.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { SavedObjectAttributes, SavedObjectReference } from 'kibana/public';
```

Comment:
Looks like this should stay `/public`

Reply:
thx, seems to be a leftover from branching from the graph migration PR, removed